src/openloco/objects/vehicle_object.h

```diff
@@ -4,6 +4,24 @@
 
 namespace openloco
 {
+    enum class vehicle_class : uint8_t
```

Comment:
vehicle_medium?

Reply:
`TransportMode`?

Reply:
`VehicleKind`?

Reply:
`TransportMode` sounds fine to me, even if it doesn't sound vehicle-exclusive. To me, `VehicleKind` sounds too close to `VehicleType`, though, which we already have.

Reply:
Yeah I think TransportMode sounds the best. I'll go for that.